### ISSUE TITLE
Unable to delete file in IDE

### ISSUE THREAD

**Issue by korzq** [codercom/code-server#151]
- `code-server` version: Can't find version number, pulled from codercom/code-server:latest
- OS Version: Linux alpha 4.15.0-46-generic x86_64 x86_64 x86_64 GNU/Linux
- Docker Version: Docker version 18.09.3, build 774a1f4

#### Steps to Reproduce

1. Start coder IDE
2. Make a file
3. Right click the file
4. Click Delete
5. In the popup menu, click move to trash
6. refresh the webpage and you see the file is back

#### Workaround

use the rm command from terminal

**Comment by creativeguy2013** [comment]
I can confirm this error. This seems to also happen when run from macOS binary.

More detailed log below:

```
ERR Cannot find module 'trash': Error: Cannot find module 'trash'
    at convert (webpack-internal:///./packages/protocol/src/common/util.ts:63:37)
    at Object.exports.parse (webpack-internal:///./packages/protocol/src/common/util.ts:83:18)
    at eval (webpack-internal:///./packages/protocol/src/browser/client.ts:158:35)
    at eval (webpack-internal:///./packages/events/src/events.ts:37:58)
    at Array.forEach (<anonymous>)
    at Emitter.emit (webpack-internal:///./packages/events/src/events.ts:37:28)
    at Client.handleMessage (webpack-internal:///./packages/protocol/src/browser/client.ts:200:36)
    at eval (webpack-internal:///./packages/protocol/src/browser/client.ts:46:23)
    at eval (webpack-internal:///./packages/events/src/events.ts:37:58)
    at Array.forEach (<anonymous>)
```

**Comment by kylecarbs** [comment]
Will fix!

**Comment by sujunmin** [comment]
Hi, I found this problem can be fixed temporarily by changing file delete in setting.
Uncheck send the deleted file to trash bin and you will see the delete option change to permanently delete, and the file will be gone when you select the permanently delete.

**Comment by korzq** [comment]
> Uncheck send the deleted file to trash bin and you will see the delete option change to permanently delete, and the file will be gone when you select the permanently delete.

Can confirm it works.

**Comment by coderasher** [comment]
Probably fixed by https://github.com/codercom/code-server/commit/e1dc6967ed1001b4e455f578e42b55496778f52d.

**Comment by sujunmin** [comment]
> Probably fixed by [e1dc696](https://github.com/codercom/code-server/commit/e1dc6967ed1001b4e455f578e42b55496778f52d).

Can confirm it works, and the Trash Bin will be at `~/.local/share/Trash/files` (debian stretch), maybe this issue could be closed.

**Comment by korzq** [comment]
confirmed in coder build at 2019-03-13 15:36:09 
closing
